Makefile

```diff
@@ -190,15 +178,11 @@ docker-buildx: generate ## Build and push docker image for the manager for cross
 	- docker buildx rm project-v3-builder
 	rm Dockerfile.cross
 
-.PHONY: podman-push
-podman-push: ## Push container image with the manager.
-	podman push ${PODMAN_PUSH_PARAMS} ${IMG}
-
 .PHONY: container-build
-container-build: test ## Build the container image
-	cekit -v --descriptor image.yaml build ${build_options} $(BUILDER)
+container-build: ## Build the container image
+	cekit -v --descriptor images/manager-image.yaml build ${build_options} $(BUILDER) --build-arg SOURCE_DATE_EPOCH="$(shell git log -1 --pretty=%ct)"
```

Comment:
Not sure if I'm doing wrong, but when executing:

make container-build BUILDER=docker IMG=quay.io/wmedvede/sonataflow-operator:latest

I get this error:


cekit -v --descriptor images/manager-image.yaml build  docker --build-arg SOURCE_DATE_EPOCH="1710355577"
Usage: cekit build docker [OPTIONS]
Try 'cekit build docker --help' for help.

Error: No such option: --build-arg
make: *** [Makefile:183: container-build] Error 2

Reply:
btw, I'm using 4.9.1

Reply:
my fault :smile: 4.11.0 is now required

Reply:
Yes :D